barretenberg/cpp/src/barretenberg/honk/proof_system/combiner.test.cpp

```diff
@@ -0,0 +1,189 @@
+#include "barretenberg/honk/flavor/ultra.hpp"
+#include "barretenberg/honk/instance/instances.hpp"
+#include "barretenberg/honk/proof_system/protogalaxy_prover.hpp"
+#include "barretenberg/honk/utils/testing.hpp"
+#include "barretenberg/proof_system/relations/relation_parameters.hpp"
+#include <gtest/gtest.h>
+
+using namespace proof_system::honk;
+namespace barretenberg::test_protogalaxy_prover {
+using Flavor = proof_system::honk::flavor::Ultra;
+using Polynomial = typename Flavor::Polynomial;
+using FF = typename Flavor::FF;
+using RelationParameters = proof_system::RelationParameters<FF>;
+
+TEST(Protogalaxy, Combiner2Instances)
```

Comment:
`CombinerOnTwoInstances` maybe? ditto on the other test

Reply:
Done